restlight-core/src/main/java/io/esastack/restlight/server/handler/Connection.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2022 OPPO ESA Stack Project
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.esastack.restlight.server.handler;
+
+import java.net.InetSocketAddress;
+import java.net.SocketAddress;
+
+/**
+ * connection.
+ */
+public interface Connection {
```

Comment:
maybe declaring a method named `id()` would be helpful to distinguish between different connections